fn_test.go

```diff
@@ -123,75 +105,49 @@ func TestRunFunction(t *testing.T) {
 					}`),
 					Observed: &fnv1.State{
 						Composite: &fnv1.Resource{
-							Resource: &structpb.Struct{
-								Fields: map[string]*structpb.Value{
-									"apiVersion": structpb.NewStringValue("s3.statnett.no/v1alpha1"),
-									"kind":       structpb.NewStringValue("Bucket"),
```

Comment:
Can you update the version and kind  in Observed of both tests, to what was used here